### PR TITLE
MRG: Py3k support

### LINKED ISSUE #143: Python 3 support
Just wanted to note that it looks like progress is being made in getting Mayavi on Python3: https://github.com/enthought/mayavi/pull/250

We should keep an eye on that and try to get PySurfer working when Mayavi stabilizes.

### PR DESCRIPTION
**Pull request by Eric89GXL**
Brings PySurfer py3k support. No CI support yet because Anaconda packages don't exist yet. But tests pass on my system with Mayavi dev and VTK 7.0.

Closes #143.

### CODE REVIEW

**Comment by agramfort** [comment]
Can you build the mne doc with it?

**Comment by Eric89GXL** [comment]
I have some annoying backend conflicts with `matplotlib` when I try to do it, but I don't think it's PySurfer's problem

File: setup.py [hunk 1]
```diff
@@ -32,10 +34,9 @@
 DOWNLOAD_URL = 'https://github.com/nipy/PySurfer'
 VERSION = version
 
-def check_dependencies():
 
-    needed_deps = ["IPython",
```
Comment (Eric89GXL) [inline setup.py]: I don't know why we had `IPython` listed as a dependency... @mwaskom looks like this was your change any idea?

**Comment by agramfort** [comment]
LGTM

File: setup.py (hunk 1 above)
Comment (mwaskom) [inline setup.py]: The `pysurfer` command line tool drops you into IPython. Also in general you need IPython to do any interactive work.

Comment (Eric89GXL) [inline setup.py]: > The pysurfer command line tool drops you into IPython.

Then it should be an optional dependency, not a hard requirement. People (like me) who do `import surfer` and do everything from there shouldn't need to have IPython.

> Also in general you need IPython to do any interactive work.

I disagree -- I use `python -i` (effectively) from Spyder in general, and rarely (if ever) use IPython, and have no problems with interactivity.

Comment (mwaskom) [inline setup.py]: Ok. I don't particularly care about declaring it as a dependency...

Comment (Eric89GXL) [inline setup.py]: Cool. Any other comments? If not I think it should be good to merge.

Comment (mwaskom) [inline setup.py]: LGTM!

**Comment by Eric89GXL** [comment]
Thanks for the quick reviews. Now everyone gets to have fun building the VTK and Mayavi :)

**Comment by mwaskom** [comment]
Forgot to say cheers for doing the boring work here!

(But also, sotto voce boos for removing my main excuse for not switching to Python 3).

**Comment by agramfort** [comment]
no pain installing mayavi on your mac michael?

**Comment by mwaskom** [comment]
That's my current supplementary excuse. I haven't actually tried to use it yet :)

**Pull request merged**